edgelet/iotedge/src/support_bundle.rs

```diff
@@ -130,33 +143,163 @@ where
         state: BundleState<M>,
         module_name: String,
     ) -> impl Future<Item = BundleState<M>, Error = Error> {
-        println!("Writing {} to file", module_name);
+        println!("Writing {} logs to file", module_name);
         let BundleState {
             runtime,
             log_options,
             include_ms_only,
+            iothub_hostname,
             file_options,
             mut zip_writer,
         } = state;
 
-        let path = Path::new("logs").join(format!("{}_log.txt", module_name));
+        let file_name = format!("{}_log.txt", module_name);
         zip_writer
-            .start_file_from_path(&path, file_options)
+            .start_file_from_path(&Path::new("logs").join(file_name), file_options)
             .into_future()
-            .map_err(|err| Error::from(err.context(ErrorKind::WriteToFile)))
+            .map_err(|err| Error::from(err.context(ErrorKind::SupportBundle)))
             .and_then(move |_| {
                 pull_logs(&runtime, &module_name, &log_options, zip_writer).map(move |zw| {
-                    println!("Wrote {} to file", module_name);
+                    println!("Wrote {} logs to file", module_name);
                     BundleState {
                         runtime,
                         log_options,
                         include_ms_only,
+                        iothub_hostname,
                         file_options,
                         zip_writer: zw,
                     }
                 })
             })
     }
+
+    fn write_edgelet_log_to_file(mut state: BundleState<M>) -> Result<BundleState<M>, Error> {
+        println!("Getting system logs for iotedge");
+        let since_time: DateTime<Utc> = DateTime::from_utc(
+            NaiveDateTime::from_timestamp(state.log_options.since().into(), 0),
+            Utc,
+        );
+        let since = since_time.format("%F %T").to_string();
+
+        #[cfg(unix)]
+        let inspect = ShellCommand::new("journalctl")
+            .args(&["-u", "-a", "iotedge"])
+            .args(&["-S", &since])
```

Comment:
You don't need two calls to `args` and one call to `arg`. A single `args` would cover all of that.